tests/ext/limiter/002-limiter-reached.phpt

```diff
@@ -24,6 +25,11 @@ while (true) {
     if ($sampled > 20) {
         break;
     }
+
+    if (--$loopBreak > 0) {
```

Comment:
Am I missing something or you wanted 
```suggestion
    if (--$loopBreak == 0) {
```

Reply:
I actually wanted a <, but apparently got the wrong key.